test/sessions.test.ts

```diff
@@ -113,3 +134,159 @@ describe("sessions.revoke", () => {
     ).resolves.toEqual({});
   });
 });
+
+/** Format the UTC timestamp truncated to second precision. */
+function iso(ts: Date): string {
+  const pad = (n: number): string => {
+    return n.toString().padStart(2, "0");
+  };
+
+  const Y = ts.getUTCFullYear();
+  const m = ts.getUTCMonth() + 1;
+  const d = ts.getUTCDate();
+  const H = ts.getUTCHours();
+  const M = ts.getUTCMinutes();
+  const S = ts.getUTCSeconds();
+  return `${Y}-${pad(m)}-${pad(d)}T${pad(H)}:${pad(M)}:${pad(S)}Z`;
+}
+
+function dateAdd(date: Date, seconds: number): Date {
+  return new Date(+date + seconds * 1000);
+}
+
+describe("sessions.authenticateJwtLocal", () => {
+  // The project ID doesn't really matter here, but this at least matches the format. Other IDs
+  // in this test come from the sandbox data.
+  const projectID = "project-test-00000000-0000-0000-0000-000000000000";
+
+  let sessions: Sessions;
+  let jwt: string;
+  let startedAt: Date;
+  let expiresAt: Date;
+
+  beforeEach(async () => {
+    // Generate a new key and add it to a local JWKS.
+    const keyID = "key0";
+    const { publicKey, privateKey } = await jose.generateKeyPair("RS256");
+    const jwk = (publicKey as crypto.KeyObject as any).export({
+      format: "jwk",
+    });
```

Comment:
Hm - when I run these tests locally I get 
```
  ● sessions.authenticateJwtLocal › zero max_token_age_seconds forces staleness

    TypeError [ERR_INVALID_OPT_VALUE]: The value "jwk" is invalid for option "format"

      192 |     const keyID = "key0";
      193 |     const { publicKey, privateKey } = await jose.generateKeyPair("RS256");
    > 194 |     const jwk = (publicKey as crypto.KeyObject as any).export({
          |                                                              ^
      195 |       format: "jwk",
      196 |     });
      197 |     const jwks = jose.createLocalJWKSet({ keys: [{ ...jwk, kid: keyID }] });

```

Reply:
Ah looks like this is a Node version difference because this got added in v15 (v16 for most production usage). Our tests run on v16. I'll add a comment for when we decide to run automated tests on older versions of Node.

For now, I've manually tested `authenticateJwt` in the latest release of each supported LTS:
- v12.22.11
- v14.19.1
- v16.14.2